bundles/org.openhab.binding.netatmo/src/main/resources/ESH-INF/thing/camera.xml

```diff
@@ -24,7 +24,30 @@
 		</channels>
 
 		<representation-property>id</representation-property>
-		<config-description-ref uri="thing-type:netatmo:nawelcomecamera"/>
+		<config-description-ref uri="thing-type:netatmo:camera"/>
+	</thing-type>
+
+	<thing-type id="NOC">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="netatmoapi" />
+		</supported-bridge-type-refs>
+
+		<label>Presence Camera</label>
+		<description>This represents a presence camera at home</description>
+
+		<channels>
+			<channel id="welcomeCameraStatus" typeId="status"></channel>
```

Comment:
It maybe will be a little bit confusing for users to have the "welcome" prefix on their channels.

Reply:
Good catch.

Reply:
CameraStatus would be better.

Reply:
Hm, I think I can't just rename it. Actual Welcome and Presence use the same API, so "Welcome" should get removed it the whole code base. When I start to rename these channels ...
1.  only for Presence to "presenceCameraStatus": I have to duplicate at least class NAWelcomeCameraHandler. The next thing is NAWelcomeCamera which comes from the swagger-api project and there are many many more naming problems like this.
2. for Welcome and Presence to "cameraStatus": It isn't downward compatible.

I see the point that it may confusing for the user that the channels are named with "welcome". It would be ok for me to realize the 1. solution but this has to stop at NAWelcomeCameraHandler. I can't refactor the whole code base with this pull-request... ;-) So I would suggest to create a new refactoring issue to resolve this. At this point at least the user doesn't recognize the naming problems and the later refactoring can also be downward compatible.

Reply:
Renaming internal classes is not the problem. It does not (really) affect a user. I do not want you to refactor all the stuff right now. We should use general naming for new channels to make them reusable. I read about a Netatmo video doorbell coming soon which will have a camera too. Touching existing channels for Welcome will be a breaking change. In my POV we can leave them untouched at the moment.

Reply:
Wouldn't renaming these channels break backwards compatibility of this binding?

Reply:
So what are we doing there ? Keep the channel names with "welcome" in the new thing type for presence camera ? Or remove welcome from the channel names only in the new presence thing typ, keeping the channel names with welcome in the welcome thing type for backward compatibility ?

Reply:
I have now pushed new changes which use "camera"... channels instead of "welcomeCamera"... channels. That required new mapping code and an inheritance structure at one class. You can decide what's the better solution, I could also revert it again. ;-)

Reply:
I would prefer if we kept backward compatibility so this change can be released in the 2.5.x branch. Otherwise this can't be merged until openHAB 3 is released.

Reply:
done